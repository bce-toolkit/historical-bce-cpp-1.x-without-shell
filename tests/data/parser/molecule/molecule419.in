C20H16N4
