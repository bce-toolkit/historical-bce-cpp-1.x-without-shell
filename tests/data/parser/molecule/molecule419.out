C 20
H 16
N 4
